lib/calecto/time.ex

```diff
@@ -38,16 +42,22 @@ defmodule Calecto.Time do
   end
 
   @doc """
-  Converts an `Ecto.Time` into a time triplet.
+  Converts an `Ecto.Time` into a time tuple.
   """
   def dump(%Calendar.Time{} = time) do
-    {:ok, Calendar.Time.to_erl(time)}
+    # TODO: Use Calendar.Time.to_micro_erl/1 if exists
+    # {:ok, Calendar.Time.to_micro_erl(time)}
+    {:ok, {time.hour, time.min, time.sec, (if time.usec, do: time.usec, else: 0)}}
```

Comment:
Good. I should probably implement the to_micro_erl funciton. Those two comment lines can stay until then.